iosApp/iosApp/Localizable.xcstrings

```diff
@@ -4896,45 +4896,8 @@
         }
       }
     },
-    "No nearby MBTA stops" : {
-      "localizations" : {
```

Comment:
suggestion (non-blocking): we should add translations for these new strings, though that can come as a separate / final sweep PR in case others snuck in too

Reply:
Added.